cleanup.php

```diff
@@ -1,51 +1,58 @@
 #!/usr/bin/env php
-
 <?php
+if (php_sapi_name() !== 'cli') {
+    http_response_code(403);
+    exit();
+}
 
-    require_once 'config.php';
-    require_once 'db_helper.php';
-    require_once 'osm_photo_note.php';
+require_once 'db_helper.php';
+require_once 'osm_photo_note.php';
 
-    $db_helper = new DBHelper();
+$db_helper = new DBHelper();
 
-    $old_inactive_photos = $db_helper->get_and_delete_old_inactive_photos();
-    foreach($old_inactive_photos as $photo) {
-        $file_name = $photo['file_id'] . $photo['file_ext'];
-        unlink($PHOTOS_TMP_DIR . '/' . $file_name);
+function deleteFile($file_path)
+{
+    echo "Deleting $file_path\n";
+    if (file_exists($file_path)) {
+        unlink($file_path);
+    }
+}
+
+$old_inactive_photos = $db_helper->getAndDeleteOldInactivePhotos();
+foreach ($old_inactive_photos as $photo) {
+    $file_path = Config::PHOTOS_TMP_DIR . DIRECTORY_SEPARATOR . $photo['file_id'] . $photo['file_ext'];
+    deleteFile($file_path);
+}
+
+$active_photos = $db_helper->getActivePhotos();
+foreach ($active_photos as $photo) {
+    $file_path = Config::PHOTOS_SRV_DIR . DIRECTORY_SEPARATOR . $photo['file_id'] . $photo['file_ext'];
+    $osm_note = new OSMPhotoNote($photo['note_id']);
+
+    if ($osm_note->http_code === 404 or $osm_note->http_code === 410) {
+        $db_helper->deletePhoto($photo['file_id']);
+        deleteFile($file_path);
     }
 
-    $active_photos = $db_helper->get_active_photos();
-    foreach($active_photos as $photo) {
-
-        $file_path = $PHOTOS_SRV_DIR . '/' . $photo['file_id'] . $photo['file_ext'];
-        $osm_note = new OSMPhotoNote($photo['note_id']);
-
-        if($osm_note->http_code == 404 or $osm_note->http_code == 410) {
-            $db_helper->delete_photo($photo['file_id']);
-            unlink($file_path);
+    if ($osm_note->http_code === 200) {
+        if (!in_array($photo['file_id'], $osm_note->photo_ids)) {
+            $db_helper->deletePhoto($photo['file_id']);
+            deleteFile($file_path);
         }
 
-        if($osm_note->http_code == 200) {
-            if(!in_array($photo['file_id'], $osm_note->photo_ids)) {
-                $db_helper->delete_photo($photo['file_id']);
-                unlink($file_path);
-            }
-
-            if($osm_note->status === 'closed'
-               and strtotime($osm_note->closed_at . ' +' . $MAX_LIFETIME_AFTER_NOTE_CLOSED_DAYS . ' days') < strtotime('now')) {
-                $db_helper->delete_photo($photo['file_id']);
-                unlink($file_path);
-            }
+        if ($osm_note->status === 'closed'
+            && strtotime($osm_note->closed_at . ' +' . (int)Config::MAX_LIFETIME_AFTER_NOTE_CLOSED_DAYS . ' days') < strtotime('now')
```

Comment:
why the `(int)` cast?

Reply:
and why changing `and` to `&&` but only here?

Reply:
Hm, it's string to time. Casting is not needed here.
I'm simply not used to seeing `and`, and I missed those in `helper.php`. Will revert it back.